specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -168,16 +194,28 @@ export function TreeRow({
                     ? 'outline outline-1 outline-blue-500'
                     : ''
                 }
+                ${
+                  isSecondFocused === row.nodeId
+                    ? 'outline outline-1 outline-green-500'
```

Comment:
does this match sp6 behavior? if not, then I don't see a reason to do it this way - instead of introducing yet another color that user has to learn and remember, why not just use a dimmed blur when not focused or gray color. OR thiker outline in active tree

that also has a benefit of better a11y (for people with color blindness)